### ISSUE TITLE
Missing context in an example?

### ISSUE THREAD

**Issue by iherman** [w3c/vc-data-model#871]
Looking at the examples in the [chapter on extensibility](https://w3c.github.io/vc-data-model/#extensibility) in the Editors' draft.  [Example 15](https://w3c.github.io/vc-data-model/#example-a-simple-credential) starts with

```json
{
    "@context ": [
        "https://www.w3.org/2018/credentials/v1",
        "https://www.w3.org/2018/credentials/examples/v1",
        "https://w3id.org/security/suites/ed25519-2020/v1"
    ],
}
```

The text then explains how one can create a new context file and this leads to [Example 17](https://w3c.github.io/vc-data-model/#example-a-verifiable-credential-with-a-custom-extension) that starts with

```json
{
    "@context": [
        "https://www.w3.org/2018/credentials/v1",
        "https://example.com/contexts/mycontext.jsonld"
    ],
}
```

The two context files `.../examples/v1` and `.../ed25519-2020/v1` are missing. One could explain the latter, because Example 17 does not include a proof section (although in my view it should, for the sake of completeness), but the former is definitely missing.

**Comment by msporny** [comment]
Agree that this should be fixed.

**Comment by msporny** [comment]
Fixed in dced606703fc36cd6a5a59c0b340e6fbfa458fdb, closing.
